package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "local-home-virual-environment",
+  "version": "0.0.1",
+  "description": "",
+  "private": true,
+  "license": "Apache-2.0",
+  "scripts": {
+    "compile": "tsc",
+    "test": "ava --verbose",
+    "lint": "eslint ."
```

Comment:
I see this project uses eslint, but I do not see an eslint config file

Reply:
Will commit eslintrc